docs/pages/install.mdx

```diff
@@ -2,44 +2,35 @@ import DownloadsComponent from "/components/DownloadsComponent";
 
 # Install Pelican
 
-This document lists the operating system requirements to install Pelican and explains how a user can download Pelican client or Pelican server components.
+This document lists the operating system requirements for Pelican and explains how a user can download and install Pelican.
+
+## Before Starting
+
+Pelican executables can run as a **client** or a **server**.
+
+For client usage, such as accessing the data, you can download and install a Pelican package or Pelican binary [following the instructions below](#install-pelican-binary).
+
+For server usage, such as serving your data to a Pelican data federation or start a new Pelican data federation, you need to run Pelican using [Pelican Docker images](./install/docker.mdx). It is not recommended to run Pelican server with a binary. Although possible, additional software dependencies and configurations are required to allow the server to be functional.
 
-> Pelican also has Docker images for users who want to serve **Pelican servers**. Refer to [Run Pelican Server using Docker image](./install/docker.mdx) for instructions.
 
 ## Supported Operating Sytems
 
-Pelican supports the following operating systems:
+Pelican supports the following operating systems, when running as a **client**:
 
 * [Red Hat Enterprise Linux (CentOS, Fedora, Alma, Rocky, or openSUSE)](./install/rhel.mdx)
 * [Debian or Ubuntu](./install/debian.mdx)
 * [Alpine Linux](./install/alpine.mdx)
-* [macOS](./install/macos.mdx) [*]
-* Windows [*]
+* [macOS](./install/macos.mdx)
+* Windows
```

Comment:
Is Windows supported only via Docker image? If so, then maybe 

```suggestion
* Windows ([via Docker](./install/docker.mdx))
```

Reply:
It's a complicated story. We do produce Windows binary and user can download it, but most of our devs don't use Windows machine so I'm not confident instructing user to do it. I'll try find a Windows machine and fill out that page.

Reply:
IIRC, we at least try to make sure Pelican is installable on Windows because the client is supposed to work across platforms. All the servers are packed into the same binary, so anyone who wants to run more than `pelican object get/put`-type commands will need the container.